tests/py2puml/conftest.py

```diff
@@ -1,5 +1,12 @@
 '''
 Fixtures that can be used by the automated unit tests.
+
+def test_using_fixtures(domain_items_by_fqn: Dict[str, UmlItem], domain_relations: List[UmlRelation]):
```

Comment:
Not sure if you ever intend to use `sphinx` or similar for building API docs, but if there's a chance, I'd suggest already following the appropriate standard: https://stackoverflow.com/questions/64451966/python-sphinx-how-to-embed-code-into-a-docstring

Reply:
No, I don't intend to use `sphinx`, I find the way to document parameter names and types redundant with good parameter names and type annotations. However, it would make sense to follow some guidelines for code snippets.

Different approaches are suggested in the stackoverflow thread you shared, which one would you recommend?

1. use the `.. code-block:: python` separator
2. use the double-column at the end of the text: `Use this module like this::`

I kind of prefer the 1st way to do it; it is a bit more verbose but it looks more explicit. The 2nd way feels like a typo to me.

Reply:
done in commit https://github.com/lucsorel/py2puml/pull/51/commits/2df8f70a50796cfe19cf574882207d713eee2bd0